### PR TITLE
57.4.0

### PR DESCRIPTION
**Pull request by ocefpaf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Should help with https://github.com/conda-forge/docutils-feedstock/pull/44

**Comment by rgommers** [comment]
Ah, fun to have old versions missing. Removing these patches to be able to build a quite old release is a little odd though. @ocefpaf is your plan to restore them straight after? If not, this may get confusing when the next 63.X release comes out.

**Comment by ocefpaf** [comment]
> Ah, fun to have old versions missing. Removing these patches to be able to build a quite old release is a little odd though. @ocefpaf is your plan to restore them straight after? If not, this may get confusing when the next 63.X release comes out.

I'm not sure I understand? This is a LTS release into a branch. We had a completely different set f patches for this release and I'm still working on them at the moment. But we do not need to restore those and this LTS branch do not affect 63.X in any way.

**Comment by rgommers** [comment]
Ah okay, I missed the custom branch the PR is made to. The diff is confusing; I'd have made one commit on top of the last commit building for `57.X`, rather than `63.X -> 57.X`. I'm not sure how to review this PR without digging into history to see if `57.X` had other patches that you are losing here with this approach.

**Comment by ocefpaf** [comment]
> Ah okay, I missed the custom branch the PR is made to. The diff is confusing; I'd have made one commit on top of the last commit building for `57.X`, rather than `63.X -> 57.X`. I'm not sure how to review this PR without digging into history to see if `57.X` had other patches that you are losing here with this approach.

Yes. That is b/c I need some other parts from modern feedstock. To stop the confusion I made this a draft so can focus on making it work first.

File: recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch [hunk 1]
```diff
@@ -19,7 +19,7 @@ index af8099a407..8f73453a6c 100644
 +            # relocatable:
 +            # https://developercommunity.visualstudio.com/comments/623156/view.html
 +            d1trimfile_opts = []
-+            if 'SRC_DIR' in os.environ and os.path.basename(self.cc) == "cl.exe":
++            if 'SRC_DIR' in os.environ:
```
Comment (isuruf) [inline recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch:22]: Why?

Comment (ocefpaf) [reply recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch:22]: This is the same patch from your PR https://github.com/conda-forge/setuptools-feedstock/pull/189.

**Comment by mattip** [comment]
Thanks!

**Pull request merged**